infra/nnfw/cmake/packages/Ruy/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ list(APPEND RUY_INCLUDES "${RuySource_DIR}")
 add_library(ruy STATIC ${RUY_SRCS})
 target_include_directories(ruy SYSTEM PUBLIC ${RUY_INCLUDES})
 target_compile_options(ruy PRIVATE -O3)
+target_compile_definitions(ruy PUBLIC USE_RUY)
```

Comment:
@ragmani I think this line should be removed.
```suggestion
```

Reply:
Thanks. I will remove it.